vnext/Desktop.IntegrationTests/WebSocketIntegrationTest.cpp

```diff
@@ -58,7 +58,7 @@ TEST_CLASS (WebSocketIntegrationTest)
     server->Start();
     string sent = "prefix";
     ws->Connect();
-    ws->Send(sent);
+    ws->Send(string{sent});
```

Comment:
>string{sent} [](start = 13, length = 12)

this should be std::move instead of copy

and on line 77, you can just store those values earlier

Reply:
Wouldn't we just be shifting the copy?
Line 77 needs the original expected value, thus, we would need to create a copy to use in such line, wouldn't we?

Reply:
But line 77 just comparing sizes, not actual strings

---
In reply to: [488193724](https://github.com/microsoft/react-native-windows/pull/5969#discussion_r488193724) [](ancestors = 488193724)